app/mailers/comment_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class CommentMailer < ApplicationMailer
+  def send_notification_email(comment_id)
```

Comment:
Assignment Branch Condition size for send_notification_email is too high. [23.54/15]